GitUI/UserControls/FilterToolBar.cs

```diff
@@ -65,6 +65,46 @@ private void ApplyCustomBranchFilter()
 
             // Apply the textbox contents, no check if the (multiple) options is in tscboBranchFilter.Items (or that the list is generated)
             string filter = tscboBranchFilter.Text == TranslatedStrings.NoResultsFound ? string.Empty : tscboBranchFilter.Text;
+            if (checkBranch && !string.IsNullOrWhiteSpace(filter))
+            {
+                List<string> newFilter = new();
+                IReadOnlyList<IGitRef> refs = _getRefs(RefsFilter.NoFilter);
+                foreach (string branch in filter.Split((char[])null, StringSplitOptions.TrimEntries | StringSplitOptions.RemoveEmptyEntries))
```

Comment:
Can you please help me understand `filter.Split((char[])null, StringSplitOptions.TrimEntries | StringSplitOptions.RemoveEmptyEntries)`?

Reply:
Like Split() with options how to split.
Null argument is all whitespace